gui/credits.h

```diff
@@ -87,6 +87,9 @@ static const char *credits[] = {
 "C1""BBVS",
 "C0""Benjamin Haisch",
 "",
+"C1""Buried",
```

Comment:
This has to be removed from this PR and then the file regenerated once it is merged.